src/adapters/worlds-manager.ts

```diff
@@ -181,6 +189,26 @@ export async function createWorldsManagerComponent({
     await database.query(sql)
   }
 
+  async function getContributableDomains(address: string): Promise<{ domains: ContributorDomain[]; count: number }> {
+    const result = await database.query<ContributorDomain>(SQL`
+      SELECT DISTINCT name, array_agg(permission) as user_permissions, size, owner
+      FROM (
+        SELECT *
+        FROM worlds w, json_each_text(w.permissions) AS perm(permission, permissionValue)
+        WHERE permission = ANY(ARRAY['deployment', 'streaming'])
```

Comment:
Do we need `streaming` here? Not sure what the UI is like, but I think we're interested in which worlds the wallet can deploy to. So I believe `deployment` only should be ok, right?